variables.tf

```diff
@@ -120,6 +120,16 @@ variable "mirror_repositories" {
       description = "Mage-OS mirror fork of https://github.com/magento/zf1."
       teams       = ["continuous-integration"]
     }
+
+    mirror-magento2-functional-testing-framework = {
```

Comment:
Just for sake of conformity, could you sort them alphabetically into the existing repositories? :see_no_evil: